demos-europe/demosplan-core
# bug (refs T28449): Add tooltip for aggregation filter items

**Ticket:** https://yaits.demos-deutschland.de/T28449

Add a tooltip with the description of the statement's processing steps. The description can be added in Verfahren -> Bearbeitungsschritte -> Beschreibung.
**Important**: This only fixes the bug in procedure -> segments. For the fix in demospipes (Aufteilen der STN) we need some backend changes first. 

### How to review/test
Add a description to one or more processing steps and go to Verfahren > Abschnitte > Filter > Schritt and hover over a step. The description should be shown in a tooltip.
